### PR TITLE
added sweetalert for a nicer alert for the delete button

### PR DESCRIPTION
**Pull request by Gazwai**
Screenshot 2022-02-01 at 02 45 40 (2)

### CODE REVIEW

**Comment by CamillieFu** [comment]
Looks so awesome!

File: app/views/dresses/listings.html.erb [hunk 1]
```diff
@@ -23,8 +23,9 @@
       <p><%= dress.description %></p>
       <p>$<%= dress.price_per_day %> per day</p>
       <div class="edit-dress">
-        <%= link_to "Delete", dress_path(dress), method: :delete, data: {confirm: "Are you sure?"}, class: "btn btn-outline-secondary mb-5 mx-1" %>
+        <%= link_to "", dress_path(dress), method: :delete, id: "delete-link", class: "hidden" %>
```
Comment (CamillieFu) [inline app/views/dresses/listings.html.erb:26]: Does the button still say delete?

File: app/javascript/packs/application.js [hunk 2]
```diff
@@ -40,7 +40,26 @@ document.addEventListener("turbolinks:load", () => {
   initMapbox();
   initAutocomplete();
 
-  $('.owl-carousel').owlCarousel({
```
Comment (CamillieFu) [inline app/javascript/packs/application.js:43]: just checking - does this code go directly into this file? not in its own file and then imported?

Comment (Gazwai) [reply app/javascript/packs/application.js:43]: I followed the Lewagon example for the sweetalert,  I don't know about the owl-carousel though. But we can ask someone about it tonight though

File: app/views/dresses/listings.html.erb (hunk 1 above)
Comment (Gazwai) [reply app/views/dresses/listings.html.erb:26]: This "link" is actually hidden and the "button" below is shown. The visible button pops up the sweetalert. When the user confirms, an event listener clicks on this hidden delete "link" to actually process the delete method.

Comment (angrysun) [reply app/views/dresses/listings.html.erb:26]: That's cool, thanks for implementing this.

**Pull request merged**